update-version

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+set -e
+
+# Whitespace-separated list of components of the form 'service|image|tag'.
+COMPONENTS="\
+    flow-manager|gpii/universal|latest \
+    preferences-dataloader|gpii/preferences-dataloader|latest \
+    preferences|gpii/universal|latest \
```

Comment:
If this is loaded from a separate file (say, components.conf), it becames more obvious what admins are expected to change. Or maybe add a comment here saying this is the only place that needs to be updated to include a new component.

Reply:
Good idea.